prow/cmd/sub/main.go

```diff
@@ -111,7 +112,7 @@ func main() {
 
 	// If we need to use a GitClient (for inrepoconfig), then we must use a
 	// InRepoConfigCache.
-	var cache *config.InRepoConfigCache
+	var cache *config.InRepoConfigCache = nil
 	var gitClientFactory git.ClientFactory
 	if flagOptions.github.TokenPath != "" || flagOptions.github.AppPrivateKeyPath != "" {
```

Comment:
cache is the only thing used downstream, I think you can move this logic to subscriber, and in that case you can do proper map[string]*config.InRepoConfigCache instead of the `AlwaysReturn` workaround

Reply:
Done. I am pretty sure this is what you had in mind